src/App.js

```diff
@@ -1,11 +1,37 @@
+// src/App.js
 import React from "react";
-import Navbar from "./Navbar/Navbar";
-import "./Navbar/Navbar.css";
+import "./App.css"; 
+import navLogo from "./images/nav-logo.jpg";
 
-export default function App() {
+function App() {
   return (
-    <div>
-      <Navbar />
+    <div className="container">
+      <nav className="navbar">
```

Comment:
Can you please create a NavBar component and put it into a folder called components